### DISCUSSION TITLE
Toggle boolean field

### DISCUSSION THREAD

**Discussion by jtsshieh** [prisma/prisma#6102] [Q&A] [answered]
Hi. Is there a way to toggle a boolean field with `update`/`upsert`? Or is there a way to do this with transactions instead or having to `find` and then `update`? Not sure if this is relevant, but I'm using PostgreSQL.

**Comment by ryands17** [comment, marked as answer]
@jtsshieh 👋 

Currently there's no way to toggle a field, so you would need to use [Optimistic Concurrency Control](https://www.prisma.io/docs/guides/prisma-guides/prisma-client-transactions-guide#optimistic-concurrency-control) to handle this case.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
